examples/modular-drivers/README.md

```diff
@@ -108,28 +110,34 @@ platform. The generated project has the form of a library crate, where
 `src/lib.rs` defines an initially empty implementation of the
 well-known Micro-RDK module entry point `register_models`:
 
-https://github.com/viam-labs/micro-rdk-esp32-sensor-examples/blob/02d7c8e487a48ac7c8d527a5c7b750d6c2357a27/src/lib.rs#L3-L5
+``` rust
+use micro_rdk::common::registry::{ComponentRegistry, RegistryError};
+
+pub fn register_models(_registry: &mut ComponentRegistry) -> anyhow::Result<(), RegistryError> {
+    Ok(())
+}
+```
 
 The generated project also includes a `package.metadata` section in
 its `Cargo.toml` which identifies the library crate as being a
 Micro-RDK module:
 
-https://github.com/viam-labs/micro-rdk-esp32-sensor-examples/blob/02d7c8e487a48ac7c8d527a5c7b750d6c2357a27/Cargo.toml#L12-L13
+https://github.com/viamrobotics/micro-rdk/blob/main/examples/modular-drivers/Cargo.toml#L18-19
```

Comment:
The old links were to specific commits so that they wouldn't become incorrect if small changes caused the lines to shift. I'd suggest making them work that way again. They can reference commit https://github.com/viamrobotics/micro-rdk/commit/fbc1783258bfefc027fd25a8cc9a1b37f6ea0524